.github/workflows/push-image.yaml

```diff
@@ -0,0 +1,49 @@
+name: Push Image
+
+on:
+  push:
+    tags: '*'
+
+env:
+  CONTAINER_REGISTRY: ghcr.io
+
+jobs:
+  build-container:
+    name: Build container image
+    runs-on: ubuntu-latest
+    env:
+      image_tag: ${{ github.repository }}:${{ github.sha }}
+
+    steps:
+      - uses: actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab
+
+      - uses: ./.github/actions/build-image
+        with:
+          image_tag: ${{ env.image_tag }}
+
+  deliver:
+    name: Build and push container image
+    needs: build-container
+    # This job is trigger only on new tags
+    if: startsWith(github.ref, 'refs/tags/')
```

Comment:
```suggestion
```
This should not be necessary, right? This workflow is triggered only with a push of a tag, so the condition is redundant

Reply:
Yes right!